### PR TITLE
run-snapd-from-snap: set the current symlink before bootstrapping

### PR DESCRIPTION
**Pull request by valentindavid**
Needed by https://github.com/snapcore/snapd/pull/13370

### CODE REVIEW

File: static/usr/lib/core/run-snapd-from-snap [hunk 1]
```diff
@@ -9,6 +9,9 @@ set -eux
 run_on_unseeded() {
     SNAPD_BASE_DIR="/run/mnt/snapd"
 
+    [ -d /snap/snapd ] || mkdir -p /snap/snapd
+    ln -sf "${SNAPD_BASE_DIR}" /snap/snapd/current
```
Comment (alfonsosanchezbeato) [inline static/usr/lib/core/run-snapd-from-snap:13]: Why is this needed now and not in the past? Maybe this should have a comment.

Comment (valentindavid) [reply static/usr/lib/core/run-snapd-from-snap:17]: I have added a comment.

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks!
